cli.js

```diff
@@ -20,7 +20,11 @@ const executeScript = (script, args) => {
   const ext = path.extname(script)
   const program = extProgramMap[ext]
   if (!program) throw new Error(`ta-script has no program assigned to handle the "${ext}" ext`)
-  cp.execSync(`${program} ${script} ${args}`, {stdio: 'inherit'})
+  try {
+    cp.execSync(`${program} ${script} ${args}`, {stdio: 'inherit'})
+  } catch (e) {
+    process.exit(1)
```

Comment:
Error will already be logged via `stdio`, but we just need to make sure to exit properly.